l1-contracts/test/foundry/unit/concrete/state-transition/chain-deps/facets/Mailbox/_Mailbox_Shared.t.sol

```diff
@@ -5,46 +5,50 @@ import {Test} from "forge-std/Test.sol";
 
 import {Utils} from "foundry-test/unit/concrete/Utils/Utils.sol";
 import {UtilsFacet} from "foundry-test/unit/concrete/Utils/UtilsFacet.sol";
-
+import {GettersFacet} from "contracts/state-transition/chain-deps/facets/Getters.sol";
 import {MailboxFacet} from "contracts/state-transition/chain-deps/facets/Mailbox.sol";
 import {Diamond} from "contracts/state-transition/libraries/Diamond.sol";
 import {IMailbox} from "contracts/state-transition/chain-interfaces/IMailbox.sol";
+import {IGetters} from "contracts/state-transition/chain-interfaces/IGetters.sol";
 import {TestnetVerifier} from "contracts/state-transition/TestnetVerifier.sol";
 
 contract MailboxTest is Test {
     IMailbox internal mailboxFacet;
     UtilsFacet internal utilsFacet;
+    IGetters internal gettersFacet;
     address sender;
-    uint256 eraChainId = 9;
+    uint256 constant eraChainId = 9;
     address internal testnetVerifier = address(new TestnetVerifier());
+    address diamondProxy;
 
-    function getMailboxSelectors() public pure returns (bytes4[] memory) {
-        bytes4[] memory selectors = new bytes4[](1);
-        selectors[0] = IMailbox.bridgehubRequestL2Transaction.selector;
-        return selectors;
-    }
-
-    function setUp() public virtual {
+    function prepare() public {
```

Comment:
This function could be named better. Something like setupDiamond or something like that which gives more context on what the function does since it is being called in multiple downstream test files.

Reply:
Fixed